dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl

```diff
@@ -317,41 +317,13 @@
 
             <!-- START CONNECT  -->
             <div class="home-col-2" style="clear: right;">
-                <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Be part of Dryad
+                <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Tweets from Dryad
```

Comment:
please change wording to "Latest from @datadryad" as per @elizabethdryad's comment.